src/main/java/me/totalfreedom/totalfreedommod/command/Command_blockcmd.java

```diff
@@ -76,8 +76,8 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         if (!playerdata.allCommandsBlocked())
         {
             playerdata.setCommandsBlocked(true);
-            FUtil.staffAction(sender.getName(), "Blocking all commands for " + player.getName(), true);
-            msg("Blocked commands for " + player.getName());
+            FUtil.staffAction(sender.getName(), "Commands for the player " + player.getName() + " have been blocked", true);
+            msg("Commands for the player " + player.getName() + " have been blocked.");
```

Comment:
Are we going for yes to full stops on return messages or no? This latest set of updates seems to have a bit of a mix and match now?

Reply:
I was told by someone in vc that having the periods on the ends of the staff action bits kinda ruins the point because its supposed to be all dramatic and stuff, so In this set of changes I changed that and I have decided to keep the messages that are only sent to the sender with the periods